src/iden3comm/constants.ts

```diff
@@ -72,3 +72,6 @@ export const SUPPORTED_PUBLIC_KEY_TYPES = {
     'JsonWebKey2020'
   ]
 };
+
+export const DEFAULT_PROOF_VERIFY_OPT = 1 * 60 * 60 * 1000; // 1 hour
```

Comment:
maybe specify in name that it's delay ?
Something like `DEFAULT_PROOF_VERIFY_DELAY_OPT`
And `DEFAULT_AUTH_VERIFY_DELAY_OPT` (or OPTS in both at the end)